backend/tests/integration/legacy/test_views.py

```diff
@@ -85,15 +85,40 @@ def test_redirects_query_from_legacy_url_with_empty_query(
 def test_redirects_query_from_legacy_url_not_in_labs(
     api_client, db, es_clear, create_record, base_app
 ):
-    with patch.dict(
-        base_app.config, {"LEGACY_BASE_URL": "https://legacy.inspirehep.net"}
-    ):
+    with patch.dict(base_app.config, {"LEGACY_BASE_URL": "https://old.inspirehep.net"}):
         response = api_client.get("/search?cc=Institutions&p=CERN&whatever=something")
 
         response_status_code = response.status_code
         response_location_header = response.headers.get("Location")
 
         expected_status_code = 302
-        expected_redirect_url = "https://legacy.inspirehep.net/search?cc=Institutions&p=CERN&whatever=something"
+        expected_redirect_url = "https://old.inspirehep.net/search?cc=Institutions&p=CERN&whatever=something"
+        assert expected_status_code == response_status_code
+        assert response_location_header == expected_redirect_url
+
+
+def test_redirects_collections_from_legacy_url(api_client, db, es_clear, create_record):
+    response = api_client.get("/collection/HepNames")
+
+    response_status_code = response.status_code
+    response_location_header = response.headers.get("Location")
+
+    expected_status_code = 301
+    expected_redirect_path = "/authors"
+    assert expected_status_code == response_status_code
+    assert response_location_header.endswith(expected_redirect_path)
```

Comment:
why no assertion on the whole string?

Reply:
Since the host was labs before, I did it like this to avoid the tests failing after releasing and redirecting labs to inspirehep.net. Yeah, now I can check the whole url.

Reply:
Indeed, as you do https://github.com/inspirehep/inspirehep/pull/859/files#diff-cca37f1084b473645317310c897326efR114

Reply:
Then the URL would depend on the environment. I guess I should use the [app_config fixture](https://github.com/inspirehep/inspirehep/blob/cdc5c7f0e21522223a2178f4388af0ad002925ed/backend/tests/integration/conftest.py#L43), right?

Reply:
Then the URL would depend on the environment. I guess I should use the [app_config fixture](https://github.com/inspirehep/inspirehep/blob/cdc5c7f0e21522223a2178f4388af0ad002925ed/backend/tests/integration/conftest.py#L43), right?

Reply:
yes exactly, it will be the same for all  tests